s3/pkg/service/object.go

```diff
@@ -135,6 +135,12 @@ func (s *s3Service) PutObject(ctx context.Context, in pb.S3_PutObjectStream) err
 	obj.TenantId = tenantId
 	log.Infof("metadata of object is:%+v\n", obj)
 	log.Infof("*********bucket:%s,key:%s,size:%d\n", obj.BucketName, obj.ObjectKey, obj.Size)
+	oldObj, err := s.MetaStorage.GetObject(ctx, obj.BucketName, obj.ObjectKey, true)
```

Comment:
Hi @hopkings2008 where the old object is deleted?

Reply:
@sfzeng when put the object with the same key used before, the underlining backend will remove the old one. the upper layer should not care about this. In s3 specification, when the put the same key, the old one will be overwritten. So, if the backend is yig, yig will responsible for removing the old one.